content/installation/webserver/WebserverAgentConfig.md

```diff
@@ -0,0 +1,85 @@
+
+
+The items that must be configured are:
+* The communication link between Contrast-Service and TeamServer
+* The NGINX service enabling the Agent to inspect traffic to certain endpoints
+
+## Configure the Contrast Service
+
+_XXX: config location is moving to /etc/contrast/webserver/contrast-security.yaml_
+
+Contrast-Service is controlled by the configuration located at `/etc/contrast/contrast_security.yaml`. This is the Contrast Common Config format that most agents utilize.  _XXX: link to a general common config document_
+
+This file will control how the Webserver is represented to TeamServer. The default configuration installed with the contrast-service linux package has most necessary items filled in however you will need to edit it with your TeamServer location and API key. You will also need to configure how you want your agent represented to TeamServer
+
+    server:
+      name: YOUR_SERVER_NAME
+      path: /
+      type: Proxy
+      environment: development
+
+The other section that needs your specific config in this file is:
+
+    contrast:
+      user_name: YOUR_CONTRAST_USERNAME
+      service_key: YOUR_CONTRAST_SERVICE_KEY
+      api_key: YOUR_CONTRAST_API_KEY
+      url: http://YOUR_TEAMSERVER_LOCATION/Contrast
+
+_XXX: where do I point someone if they are confused as to where to get these item?_
+_XXX: give the user a way to test their configuration is working wrt to TeamServer?_
```

Comment:
@palmergs Any suggestions?

Reply:
For the latter item, I can tell the user to check the logs.  Wouldn't it be cool if the TS UI just sorta told us that the agent was communicating fine.  Maybe this is a fault of the agent in that it needs to send an initial 'registration' or 'ping' msg on startup.

In the short term I'll just highlight the log file to the user and tell them to check it if there are errors.